wsl-pro-service/internal/system/system_test.go

```diff
@@ -585,6 +649,119 @@ func TestLandscapeDisable(t *testing.T) {
 	}
 }
 
+func TestEnsureValidLandscapeConfig(t *testing.T) {
+	t.Parallel()
+
+	testCases := map[string]struct {
+		systemLandscapeConfigFile string
+
+		breakWriteConfig        bool
+		breakLandscapeConfigCmd bool
+		breakWSLPath            bool
+		breakWSLDistroName      bool
+		noLandscapeGroup        bool
+
+		wantNoLandscapeConfig    bool
+		wantNoLandscapeConfigCmd bool
+		wantErr                  bool
+	}{
+		"Appends any required fields - no ssl":   {systemLandscapeConfigFile: "minimal.conf"},
+		"Transform Windows SSL certificate path": {systemLandscapeConfigFile: "windows_ssl_only.conf"},
+		"Refresh computer_title if changed":      {systemLandscapeConfigFile: "old_computer_title.conf"},
+
+		"Regular with additional keys":            {systemLandscapeConfigFile: "regular.conf"},
+		"Do not modify other sections and keys":   {systemLandscapeConfigFile: "regular_with_extra_keys.conf"},
+		"Reformat landscape config to proper ini": {systemLandscapeConfigFile: "regular_with_weird_format.conf"},
+
+		"Do not rerun landscape without modifications":                             {systemLandscapeConfigFile: "no_change_needed.conf", wantNoLandscapeConfigCmd: true},
+		"No landscape configuration means no landscape command nor config created": {systemLandscapeConfigFile: "-", wantNoLandscapeConfigCmd: true, wantNoLandscapeConfig: true},
```

Comment:
Ditto for "Landscape" capitalization :p

Reply:
(Note, I only did it for the configuration, I kept the user and command as `landscape` in that context)